docs/source/api_reference/data_format.rst

```diff
@@ -0,0 +1,273 @@
+.. _data_format:
+
+File Format Specifications
+==========================
+
+.. toctree::
+    :maxdepth: 1
+    :hidden: false
+
+    Overview
+    Introduction
+    Description
+    Instructions
+    Illustration
+
+Overview
+--------
+This document has the following content:
+
+- Introduction: What is a ``.ts`` file, why and when to use it.
+- Description: What are the individual components of a ``.ts`` file.
+- Instructions: How to create your own ``.ts`` file.
+- Illustrations: A running example to tie up above sections.
+
+Introduction
+------------
+This document formalizes string identifiers used in ``.ts`` file format.
+Encoded in ``utf-8``, ``.ts`` files stores time-series dataset and its corresponding metadata and can be
+opened via any basic editor like notepad for visual inspection.
+String identifiers refer to strings beginning with ``@`` in the file.
+
+``.ts`` files contains information blocks in the following order:
+
+1. All the comments to describe the dataset. (begins with ``#`` in the file)
+2. All the String Identifiers to give information about the metadata (begins with ``@`` in the file).
+3. Actual dataset.
+
+See `Basic Motion.ts`_ for a concrete example.
+
+These identifiers serve the purpose of containing metadata for the dataset.
```

Comment:
"these" refers to content above, but there are no identifiers above.

You also need to explain the meaning of "identifier".

Also, it is unclear why you are talking about them now. Should this not be in a section about the metadata block? Which should be preceded by a subsection about the description block?

Reply:
You're right, I've moved this under the "Metadata Block" subsection under "Description".